scripts/paquette2007.json

```diff
@@ -0,0 +1,49 @@
+{
+    "citation": "Alain Paquette, Etienne Lalibert\u00e9, Andr\u00e9 Bouchard, Sylvie de Blois, Pierre Legendre, and Jacques Brisson. 2007. Lac Croche understory vegetation data set (1998\u20132006). Ecology 88:3209.",
```

Comment:
Alain Paquette, Etienne Laliberté, André Bouchard, Sylvie de Blois, Pierre Legendre, and Jacques Brisson. 2007. Lac Croche understory vegetation data set (1998-2006). Ecology 88:3209.